src/index.ts

```diff
@@ -0,0 +1,27 @@
+import express from 'express';
+import path from 'path';
+
+import { router } from './routers/user-router';
+const app = express();
+
+app.set('port', (process.env.PORT || 4200));
+
+app.use(express.static(path.join(__dirname, '../src')));
+app.use(express.json());
+
+app.use((req, res, next) => {
+    res.header('Access-Control-Allow-Origin', '*');
+    res.header('Access-Control-Allow-Methods', 'GET, PUT, POST, DELETE');
+    res.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept, Authorization');
+    next();
+});
+
+app.use('/', router);
+app.use((req, res) => {
+    res.status(500).send('Smth went wrong!!!');
```

Comment:
I do not really understand why need that code. Could you explain me, please?

Reply:
To let the user know if the server crashes

Reply:
mm. Now you replaced behavior and we getting 500 error every time if resource non found
![image](https://user-images.githubusercontent.com/22756871/113393173-f33e6300-939e-11eb-8f61-d2eed3c773d6.png)